app/src/main/java/vn/edu/usth/minigh/IssueActivity.java

```diff
@@ -54,9 +55,13 @@ public void addFrag(String txt, int number){
         ft.commit();
     }
 
-    public void goToIssue(View view) {
-        Intent intent = new Intent(this, IssueDiscussionActivity.class);
+    public void goToDiscuss(View view) {
+        Intent intent = new Intent(this, DiscussionActivity.class);
         intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Hah I missed this when y'all run the night.  IIUC [this is not something we would want in this scenario](https://developer.android.com/reference/android/content/Intent#FLAG_ACTIVITY_NEW_TASK).

Reply:
It's should not be used when we need the response from another activity? Is that? Or I miss something?

Reply:
I don't get what you mean by *response from another activity*.  I was referring to that we don't want to give the discussion activity a new back stack.